supervision/annotators/core.py

```diff
@@ -1609,7 +1609,7 @@ def __init__(
         position: Position = Position.CENTER,
         trace_length: int = 30,
         thickness: int = 2,
-        color_lookup: ColorLookup = ColorLookup.CLASS,
+        color_lookup: ColorLookup = ColorLookup.INDEX,
```

Comment:
Class IDs are not necessary, but when they are present (in most cases), it's worth showing the trails corresponding to class colors. Let's keep this as `.CLASS`.

Reply:
But we removed class ids argument, so now we don't have it. If you want to keep color lookup bytetrack should work like my previous PR

Reply:
Not all situations that use a tracker will use `TraceAnnotator`. We enabled tracker for those people, but let's not break the defaults of `TraceAnnotator` - let's not break the defaults.

`resolve_color` will work as before as it takes entire detections, regardless if color was passed to ByteTrack. In 90% of the cases, detections will have class ID. In cases where they don't, `resolve_color_idx` will tell users why it's failing and how to fix it.

At least that's how I remember it